### ISSUE TITLE
Accessibility:List: The “multi selectable” information about list items is not read when accessed

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3478]
Description: 
Case 1:
The “multi selectable” information about list items is not read when accessed. 
Case 2:
Information about radio buttons/selectable information about list items is not conveyed to the user, when list items are accessed. 

Expected: The information about checkboxes and radio buttons (single selection and multi selection) should be conveyed to the user

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94647762-cc7b7300-030e-11eb-89a7-de0738c606f7.png)

![image](https://user-images.githubusercontent.com/32538291/94647772-d1402700-030e-11eb-8cec-2c0219962a47.png)

![image](https://user-images.githubusercontent.com/32538291/94647785-d9986200-030e-11eb-81dd-3fb59d9a71b6.png)
